vllm/executor/ray_gpu_executor.py

```diff
@@ -171,23 +173,17 @@ def _init_workers_ray(self, placement_group: "PlacementGroup",
                           max_concurrent_workers=self.parallel_config.
                           max_parallel_loading_workers)
 
-    def execute_model(
+    def _driver_execute_model(
```

Comment:
Can you add a docstring executing this with an empty input will stop worker execution loop?